testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
```

Comment:
If you think you are done these tests should be included into the main gro test file